app/initializers/export-application-global.js

```diff
@@ -1,7 +1,8 @@
 import Ember from 'ember';
 import config from '../config/environment';
 
-export function initialize(container, application) {
+export function initialize() {
+  var application = arguments[1] || arguments[0];
```

Comment:
@mike-north - Are you saying that the initializers that are internal to Ember itself are triggering this deprecation?  Can you open an issue for that (the fix in Ember itself isn't quite like this though)?